zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/SelectAutocompleteValues.java

```diff
@@ -65,6 +65,10 @@
       .setInt("limit_", 1000)); // no one is ever going to browse so many tag values
   }
 
+  @Override public ResultSet map(ResultSet input) {
```

Comment:
these have to return result set for follow-up requests. a later PR will likely abstract asynchronous pagination